stacks/spica/src/passport/pages/apikey-add/apikey-add.component.ts

```diff
@@ -0,0 +1,49 @@
+import {Component, OnInit, TemplateRef, ViewChild, OnDestroy} from "@angular/core";
+import {ApiKey, emptyApiKey} from "src/passport/interfaces/apikey";
+import {Router, ActivatedRoute} from "@angular/router";
+import {filter, switchMap, takeUntil} from "rxjs/operators";
+import {apiKeyService} from "src/passport/services/apikey.service";
+import {Subject} from "rxjs";
+
+@Component({
+  selector: "app-apikey-add",
+  templateUrl: "./apikey-add.component.html",
+  styleUrls: ["./apikey-add.component.scss"]
+})
+export class ApiKeyAddComponent implements OnInit, OnDestroy {
+  @ViewChild("toolbar", {static: true}) toolbar: TemplateRef<any>;
+  public apiKey: ApiKey = emptyApiKey();
+
+  private onDestroy: Subject<void> = new Subject<void>();
+
+  constructor(
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private apiKeyService: apiKeyService
```

Comment:
`private apiKeyService: apiKeyService` should be `private apiKeyService: ApiKeyService`.

You need to rename it to ApiKeyService.